guides/lambda/README.md

```diff
@@ -0,0 +1,57 @@
+# How to Set Up a Lambda
+
+Lambdas require some additional set up.
+
+## Creating the Lambda in AWS
+
+When you create a lambda that will be deployed with uDeploy you need to add versioning and create an alias for each environment.  The alias must point to a specific version.  If you point to $LATEST uDeploy will not work.  Additionally, you must use the lambda description field to hold your semantic version number.  
+
+### Terraform 
+
+To create an alias that points to a specifc version:
```

Comment:
Should we explain that whatever AWS service is calling the lambda function should now call the function alias allowing udeploy to control which version of code is being used?